packages/app/app/stacks/stacks-orders.tsx

```diff
@@ -0,0 +1,112 @@
+"use client";
+
+import {
+  StackOrder,
+  filterActiveOrders,
+  filterCompletedOrders,
+  getStackOrders,
+} from "@/models/stack-order";
+import { Tab } from "@headlessui/react";
+import { BodyText } from "@/ui";
+import { StacksTable } from "@/components/StacksTable";
+import { EmptyState } from "@/app/stacks/empty-state";
+import { getOrders } from "@/models/order";
+import { ChainId } from "@stackly/sdk";
+import { useEffect, useState } from "react";
+import { cx } from "class-variance-authority";
+
+export interface StackOrdersProps {
+  address: string;
+  chainId: ChainId;
+}
+
+export const StackOrders = ({ chainId, address }: StackOrdersProps) => {
+  const [loading, setLoading] = useState(true);
+  const [completedOrders, setCompletedOrders] = useState<StackOrder[]>([]);
+  const [activeOrders, setActiveOrders] = useState<StackOrder[]>([]);
+  const [currentStackOrders, setCurrentStackOrders] = useState<StackOrder[]>(
+    []
+  );
```

Comment:
There is no necessity for this currentStackOrders. We can populate active and complete orders directly and there is no need for an extra `useEffect`. To check the empty state we just need to check if both active and complete orders are empty.

Reply:
we can have just currentStackOrders and not the active/complete orders on state

Reply:
Nice, that also works